lua/telescope/pickers.lua

```diff
@@ -723,6 +723,14 @@ function Picker:set_selection(row)
       0,
```

Comment:
Let's add this highlight not starting from 0 anymore (since we're just going to overwrite immediately after.